docs/content/guides/6.multistore/3.patterns/2.design/4.different-code-per-store.md

```diff
@@ -5,4 +5,254 @@ navigation:
   icon: tabler:number-1-small
 ---
 
-# Different markups for individual components
+# Different Markups for Individual Components
+
+## Prerequisites
+
+Familiarity with [Tooling & Concepts → File-Based Inheritance](/guides/multistore/tooling-and-concepts/file-based-inheritance) is required to understand file override mechanisms and inheritance principles.
+
+**What You’ll Learn**
+
+::list{type="success"}
+- How to safely modify the markup of individual components in a multibrand project without breaking their props interfaces
+- How to handle breaking changes in component props with step-by-step workflows, including duplicating and modifying files
+- Best practices for managing component markup customization to maintain consistency and avoid duplication
+- How to implement these changes using examples in Next.js and Nuxt to adapt to specific frameworks
+::
+
+## Why Customize Component Markups?
+
+In multibrand setups, different stores often require unique layouts or styling. Customization is necessary to ensure proper alignment with branding and design guidelines while sharing common functionalities across stores. For example:
+
+- Store-specific customizations to match unique branding designs.
+- Adjustments to the structure or layout of a component in one brand without affecting others.
+
+Changing a component’s markup is a **structured process**, and this guide will walk you through **safe modifications** and scenarios where **breaking changes** are unavoidable.
+
+## Safe Markup Changes Without Breaking Props
+
+The most straightforward way to customize a component is by modifying **only its markup** while keeping its props interface unchanged. This ensures compatibility across the codebase and minimizes any disruption to dependent components or features.
+
+### When to Use
+
+- To rearrange the component structure.
+- To adjust styling for specific stores or brands without altering the functionality.
+
+### The Solution: Component Override
+
+In cases where a specific store requires customization, you can override the component by duplicating and modifying it within that store's directory.
+
+#### Example Workflow
+
+1. **Duplicate** the component inside the store-specific directory.
```

Comment:
What do you think about adding an example bash `cp` command here?

Reply:
Will it be helpful for AI? I've added it.

Reply:
definietely